packages/api/src/store/db.ts

```diff
@@ -103,6 +106,13 @@ export class DB {
       console.log("no replica url found, not using read replica");
     }
 
+    this.metricHistogram = new Histogram({
+      name: "livepeer_api_pgsql_query_duration_seconds",
+      help: "duration histogram of pgsql queries",
+      buckets: [0.003, 0.03, 0.1, 0.3, 1.5, 10],
+      registers: [register],
```

Comment:
Why does this need a register and the segment one doesn't? Also don't we need to specify `labelNames` here as well?

Reply:
its the default register. if we're going to have multiple registries, we would need this. Can be removed here. and yes, I forgot to stage the line with `labelNames` field here.